New Changelog: More love for innersource collaboration and enterprise fork policies (#165)
yuhattor/hattori.dev: content/changelog/2023-02-16-more-love-for-innersource-collaboration-and-enterprise-fork-policies.md

---

> title: "インナーソースコラボレーションとエンタープライズフォークポリシーにもっと愛を込めて"

Reviewer: ```suggestion
title: "インナーソースコラボレーションとエンタープライズフォークポリシーに愛を込めて"
```

author: "Kevin Duck"
description: " In June 2022 we updated fork capabilities to include forking a repository into the same organization as its upstream repository, forking internal repositories to enterprise organizations, and for enterprise owners to limit where forks can be created. This opened up a lot of new possibilities for collaboration!  We recently updated fork capabilities again to unblock an additional workflow: fork repositories into another organization more than once. Before, when you tried to fork a repository into another organization that already had a fork of that repository, your option to finish forking into that organization was grayed out and GitHub let you know that a fork already exists in the target organization. With this update, you will have the option to continue forking it using a unique name.  "
coverimage: "https://i0.wp.com/user-images.githubusercontent.com/5758031/218177396-a89b3749-c064-4e28-b931-1ab2fc2bb6f4.png?ssl=1"
englishsummary: "  GitHub recently updated fork capabilities to allow for forking a repository into another organization more than once, unlocking new possibilities for collaboration."

> summary: "  GitHubは最近、フォーク機能を更新し、リポジトリを別の組織に複数回フォークできるようになり、コラボレーションの新しい可能性が開かれました。"

Reviewer: ```suggestion
summary: "GitHub は最近、フォーク機能を更新し、リポジトリを別の組織に複数回フォークできるようになり、コラボレーションの新しい可能性が開かれました。"
```

---

> <p>2022年6月、私たちはフォーク機能を<a href="https://github.blog/changelog/2022-06-27-improved-innersource-collaboration-and-enterprise-fork-policies">アップデート</a>し、上流のリポジトリと同じ組織にリポジトリをフォークしたり、内部リポジトリを企業組織にフォークしたり、企業オーナーがフォークの作成場所を制限したりできるようにしました。これにより、コラボレーションの新しい可能性が広がりました。</p>

Reviewer: ```suggestion
<p>2022年6月、私たちはフォーク機能を<a href="https://github.blog/changelog/2022-06-27-improved-innersource-collaboration-and-enterprise-fork-policies">アップデート</a>し、アップストリームのリポジトリと同じ組織にリポジトリをフォークしたり、インターナルリポジトリを企業組織にフォークしたり、エンタープライズオーナーがフォークの作成場所を制限したりできるようにしました。これにより、コラボレーションの新しい可能性が広がりました。</p>
```

> <p>私たちは最近、フォーク機能を再び更新し、追加のワークフローをブロックしないようにしました：リポジトリを別の組織に複数回フォークすることです。以前は、すでにそのリポジトリをフォークしている別の組織にリポジトリをフォークしようとすると、その組織へのフォークを終了するオプションがグレーアウトし、ターゲット組織にすでにフォークが存在することを GitHub が知らせました。今回のアップデートでは、ユニークな名前を使ってフォークを継続するオプションがあります。</p>

Reviewer: ```suggestion
<p>私たちは最近フォーク機能をアップデートし、追加のワークフローをブロックしないようにしました: リポジトリを別の組織に複数回フォークすることです。以前は、すでにそのリポジトリをフォークしている別の組織にリポジトリをフォークしようとすると、その組織へのフォークを終了するオプションがグレーアウトし、GitHub がターゲット組織にすでにフォークが存在することを知らせます。今回のアップデートでは、ユニークな名前を使ってフォークを継続するオプションがあります。</p>
```
